kube/services/selenium/selenium-hub-deployment.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: selenium-hub
+  name: selenium-hub
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: selenium-hub
+  template:
+    metadata:
+      labels:
+        app: selenium-hub
+    spec:
+      containers:
+      - image: selenium/hub:3.141
```

Comment:
change this to come from the manifest

Reply:
The `kube-setup-seleniumhub` is a TBD, so we will add this improvement in a subsequent PR.
It should be a good onboarding task.